scripts/asm-installer/install_asm

```diff
@@ -96,6 +97,9 @@ USE_HUB_WIP=0
 ENVIRON_PROJECT_ID=""
 HUB_MEMBERSHIP_ID=""
 
+# Environment variable used by VM
+VM_EXPANSION=0
```

Comment:
Can you name it USE_ to be the same as Hub WIP? USE_VMS maybe

Reply:
Done. Renamed to USE_VM. VMS sometimes has other representation.